Added details about adding custom sections to resume

syedzubeen opened the pull request.

syedzubeen committed:
Added details about adding custom sections to the resume

syedzubeen committed:
Added asset image for custom section doc updates

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/resuminator/docs?pullRequest=24) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/resuminator/docs?pullRequest=24) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/resuminator/docs?pullRequest=24) it.</sub>

syedzubeen committed:
Fixed linting issues

syedzubeen commented:
Hi @merrcury,

Added a small section and did some updates, also fixed all linting issues with the commit, yet the deploy preview is failing. Could you help me out with what I am missing here?
localhost also seems to be working without any issues.

merrcury commented:
> Hi @merrcury,
> 
> Added a small section and did some updates, also fixed all linting issues with the commit, yet the deploy preview is failing. Could you help me out with what I am missing here? localhost also seems to be working without any issues.

Hi @syedzubeen Looks like the file you have added has the name `create_new_section.PNG` while in the documentation you mentioned `create_new_section.png`

I would recommend you change the file name to `create_new_section.png`

syedzubeen committed:
Delete create_new_section.PNG

syedzubeen committed:
Added file with correct extension

merrcury reviewed (approved):
Great Job @syedzubeen
